index.html

```diff
@@ -55,22 +55,22 @@ <h1 class="headline__title">I’m Ahmed H. Glad to see you!</h1>
     <ul class="headline__icons">
       <li>
         <a href="#">
```

Comment:
- [ ] **Required:** Most of your anchor tags are set to your root `<a href="#">`. Kindly check your anchor tags for those that would not point to another page. Let's remove them, these would greatly improve the accessibility of your website.